From 37a7a22d3f9d786c85cb92f5377f93a352b9b8df Mon Sep 17 00:00:00 2001
From: Dinh Nguyen <Dinh.Nguyen@freescale.com>
Date: Tue, 26 Feb 2008 08:12:12 -0600
Subject: [PATCH] ENGR00066530 Fix export of ioremap and iounmap functions

Because of a previous checkin, __mxc_ioremap and __mxc_iounmap were
not exported, which caused instability in the USB host driver on
MX31.

Signed-off-by: Dinh Nguyen <Dinh.Nguyen@freescale.com>
---
 include/asm-arm/arch-mxc/io.h |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/include/asm-arm/arch-mxc/io.h b/include/asm-arm/arch-mxc/io.h
index 00c84ec..f25eb75 100644
--- a/include/asm-arm/arch-mxc/io.h
+++ b/include/asm-arm/arch-mxc/io.h
@@ -23,6 +23,13 @@
 /*! Allow IO space to be anywhere in the memory */
 #define IO_SPACE_LIMIT 0xffffffff
 
+extern void __iomem *__mxc_ioremap(unsigned long cookie, size_t size,
+				unsigned int mtype);
+extern void __mxc_iounmap(void __iomem *addr);
+
+#define __arch_ioremap(a, s, f) __mxc_ioremap(a, s, f)
+#define __arch_iounmap(a)        __mxc_iounmap(a)
+
 /*!
  * io address mapping macro
  */
-- 
1.5.4.4

